webpack.config.js

```diff
@@ -0,0 +1,31 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: './src/index.js',
+  output: {
+    filename: 'main.js',
+    path: path.resolve(__dirname, 'dist'),
+    clean: true,
+  },
+  devServer: {
+    static: './dist',
+  },
+  module: {
+    rules: [
+      {
+        test: /\.css$/i,
+        use: ['style-loader', 'css-loader'],
+      },
+    ],
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
+  ],
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/201902201-65794b6a-dedf-47b2-b74a-7cb622f9a67e.png)

- [ ] Your application is not starting properly. :fearful: KIndly fix this issue as shown in the image above :arrow_up:

Reply:
@HAMMAS-SALEEM  Thank you for the suggestion